src/sagas/wallet.js

```diff
@@ -219,10 +231,12 @@ export function* startWallet(action) {
   yield fork(listenForWalletReady, wallet);
 
   try {
-    const serverInfo = yield call(wallet.start.bind(wallet), {
+    console.log('[*] Start wallet.');
```

Comment:
Remove?

Reply:
Maybe, this one was a good sanity test while running the wallet.
I had some issues with irresponsive fullnodes and unleash not responding, with access to the console we can check both that we requested the start wallet and that we have connected to the correct network.